src/components/order-service/order-service.controller.ts

```diff
@@ -313,6 +351,14 @@ export class OrderServiceController implements angular.IController {
     }
   }
 
+  private updateBindParametersStepVisibility() {
+    // Show the bind parameters step if the bind step if not hidden and the plan has a bind parameter schema.
+    this.bindParametersStep.hidden = this.bindStep.hidden ||
+                                     this.ctrl.bindType === 'none' ||
+                                     !_.has(this.ctrl, 'bindParameterSchema.properties');
+    this.bindParametersStep.allowed = !this.bindParametersStep.hidden && this.bindStep.valid;
```

Comment:
nit: don't really need to check hidden here, if it is hidden then allowed will never be used.